app/models/flood_risk_engine/address_search.rb

```diff
@@ -0,0 +1,5 @@
+module FloodRiskEngine
+  class AddressSearch < ActiveRecord::Base
```

Comment:
no worries, I was pretty much making it up as I went ;-)  .. seemed like a nice colour